ksql-cli/src/main/java/io/confluent/ksql/cli/console/JLineReader.java

```diff
@@ -124,7 +127,10 @@ public String expandVar(final String word) {
 
   @Override
   public String readLine() throws IOException {
-    final String line = lineReader.readLine(prompt);
+    final String line = lineReader
+        .readLine(prompt)
+        .replace("\n", "");
```

Comment:
Replace "" with " ". Without space the two lines will be concatenated without any space in between.

Reply:
We wouldn't want a space to be introduced though.  For example, this is totally valid:

```
SELECT * FR
OM Blah;
```

Reply:
To put it another way...

If we add a space then the user is limited in how they can split statements over multiple lines. If we don't, then the user gets to chose if there should be space or not. Power to the user!